pages/api/ipfs/types.ts

```diff
@@ -0,0 +1,18 @@
+import * as z from "zod";
+
+export const IOMarketMetadata = z.object({
```

Comment:
Might be nice to share with market form validation and add some rough size restrictions if possible